desktop/src/com/limegroup/gnutella/gui/menu/HelpMenu.java

```diff
@@ -67,7 +65,7 @@
 
         addMenuItem(new OpenLinkAction("http://twitter.com/#!/frostwire", I18n.tr("Follow us on Twitter"), I18n.tr("Follow us on Twitter")));
 
-        addMenuItem(new OpenLinkAction("http://www.frostwire.com/forum/viewforum.php?f=1", I18n.tr("Foru&m"), I18n.tr("Access the FrostWire Users\' Forum")));
+        addMenuItem(new OpenLinkAction("http://forum.frostwire.com/", I18n.tr("Foru&m"), I18n.tr("Access the FrostWire Users\' Forum")));
```

Comment:
Please append this to the URL: `?from=desktop-help` -> `http://forum.frostwire.com/?from=desktop-help`